steering_controllers_library/src/steering_odometry.cpp

```diff
@@ -133,11 +133,17 @@ double SteeringOdometry::get_linear_velocity_double_traction_axle(
   const double steer_pos)
 {
   double turning_radius = wheelbase_ / std::tan(steer_pos);
+  double vel_wheel_r = right_traction_wheel_vel * wheel_radius_;
+  double vel_wheel_l = left_traction_wheel_vel * wheel_radius_;
+
+  if (std::isinf(turning_radius))
```

Comment:
Will we need to handle cases of NaN as well here?

Reply:
I am not sure; there are already input checks here: https://github.com/ros-controls/ros2_controllers/blob/c08bdab049cc090df0643d3399bf53862f4571be/tricycle_steering_controller/src/tricycle_steering_controller.cpp#L68 . I can update the PR if desired but what should be the behavior when the input is not finite, raise an exception?

Reply:
If the checks are already done. It is fine. Thank you!